app/api/resources/user.py

```diff
@@ -337,7 +337,7 @@ def post(cls):
 
         if result[1] is HTTPStatus.CREATED:
             send_email_verification_message(data["name"], data["email"])
-
+        print(result[0])
```

Comment:
😝 can you remove this print() statement?

Reply:
```suggestion
```
Or is it needed @mtreacy002 ?

Reply:
if it's not needed, just remove it and I am happy to merge it after @mtreacy002

Reply:
Ooppss, forgot to take it out 😂. Will remove it asap, @isabelcosta 😉

Reply:
Done, @isabelcosta , please review again when you have time 😉